File: 7/Problem3.c
#include<stdio.h>

void DisplayEvenFactorial(int iNo)
{
    int iCnt = 0;
    int iFact = 1;
    if(iNo < 0)
    {
        iNo = -iNo;
    }
    for(iCnt = 2; iCnt <= iNo  ; iCnt += 2)
    {
        iFact = iFact * iCnt;
    }
    
    printf("Factorial is : %d\n",iFact);
    
}
int main()
{
    int iValue = 0;
     printf("Enter the number : ");
     scanf("%d",&iValue);

     DisplayEvenFactorial(iValue);

     return 0;
}

File: 21/problem3.java
import java.util.*;
class DigitX
{
    public static void DisplayFactors(int iNo)
    {
        int iCnt = 0;
        for( iCnt = 1; iCnt <= iNo/2; iCnt++)
        {
            if(iNo % iCnt == 0)
            {
                System.out.print(iCnt + " \n");
            }
        }
    }
}


class problem3
{
    public static void main(String args[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter the number : ");
        iValue = sobj.nextInt();

        DigitX.DisplayFactors(iValue);
    }
}